src/nn.h

```diff
@@ -32,6 +32,17 @@ extern "C" {
 #include <errno.h>
 #include <stddef.h>
 
+/*  64-bit integer type for statistics.  */
+#if defined(_WIN32)
+typedef	 __uint64 NN_UINT64
```

Comment:
Win7 works perfectly fine even thru the latest-greatest MSVC 2015. Speaking as one stakeholder on this platform, if MSVC 2010 is the lowest common denominator for this particular feature (among perhaps others), I find it perfectly acceptable to make >=2010 the cutoff.

WinXP -- I cannot speak for, because my business officially does not support this. It's >2yrs out of Microsoft's support at this point.

Feel free to put out a call whether versions *prior* to MSVC 2010 should still be supported, but from my point of view, there should be no problem making >=2010 a requirement.